primefaces/src/main/java/org/primefaces/component/tabview/TabBase.java

```diff
@@ -39,7 +39,8 @@ public enum PropertyKeys {
         closable,
         titletip,
         ariaLabel,
-        menuTitle
+        menuTitle,
+        id
```

Comment:
I dont think thats a good idea, its inherited actually from UIComponent and autogenerated by facelets if not set

Reply:
I was going off this part:
<img width="1161" alt="Screenshot 2023-12-30 at 5 07 54 AM" src="https://github.com/primefaces/primefaces/assets/35503797/7e3b6e6b-4a8a-4fb8-ad49-e2a8b79259fc">

The `li` element did not get any `id`. What do you think would be a better approach?

Reply:
Agreed remove this and use `tab.getClientId(context)`

Reply:
Done!